package.json

```diff
@@ -80,7 +80,7 @@
     "axios": "^0.19.0",
     "babel-core": "6",
     "babel-loader": "7",
-    "babel-plugin-istanbul": "^5.2.0",
+    "babel-plugin-istanbul": "4.1.6",
```

Comment:
Since this is a version downgrade, we should probably note what the issue is so we can know when/if we can return to current versions.  

Putting that note here sounds sufficient.  What is the error this fixes and why was a downgrade needed?

Reply:
From what I saw in the PR linked, this is just an incompatibility with babel where widget is using babel 6 but `plugin-istanbul version 5` is using babel 7.

Reply:
yeah, it's exact what Nikhil mentioned.